### PR TITLE
Fixed sytnax errors in readme

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ $ yarn add --dev native-ext-loader
 
 Update rules entry in the Webpack configuration file:
 
-```javascript
+```js
```
Comment (smt116) [inline README.md:21]: I believe that `javascript` is [a valid mode](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2143-L2178) for the code snippets. Did you find other information?

**Review by smt116 — CHANGES_REQUESTED** [review]
Please reword the commit message to follow [the common rule of present tense](https://gist.github.com/smt116/df307ad726f69b40f205b0ac68badc96#a-properly-formed-git-commit-subject-line-should-always-be-able-to-complete-the-following-sentence).

File: README.md (hunk 1 above)
Comment (amilajack) [reply README.md:21]: `javascript` works. `js` is more concise and is the convention

Comment (smt116) [reply README.md:21]: Fair enough. Please change it also in the code snippet below.

**Comment by smt116** [comment]
@amilajack Please fix the https://github.com/smt116/node-native-ext-loader/pull/2#pullrequestreview-113155837 and it will be good to merge. Thanks for the pull request, 👍.

**Comment by smt116** [comment]
@amilajack I've fixed the commit message manually and [merged the code](https://github.com/smt116/node-native-ext-loader/commit/5886563889c6beeaa83a3e46b3233f7728968965) via the console. Thanks for the pull request!

**Pull request closed without merging**